lib/compat/wordpress-6.0/theme.json

```diff
@@ -240,6 +241,8 @@
 		}
 	},
 	"styles": {
-		"spacing": { "blockGap": "24px" }
+		"spacing": {
+			"blockGap": "24px"
```

Comment:
I removed the default padding from here as it would impact all themes, which would be a problem.